frontend/src/routes/onboard/onboard.tsx

```diff
@@ -64,7 +70,7 @@ export function OnboardHome() {
         // Add wallets and networks to state
         dispatch({ type: 'ADD_WALLETS', wallets: onboarding.wallets })
       } else {
-        navigate(OnboardPaths.WalletCreate)
+        navigate('/onboard/wallet-create')
```

Comment:
follow up (non-blocking): we use route names on all our other apps so these don't get jumped with path reordering. Perhaps we should do that here as well?

Reply:
Lets follow up!